src/app/search/search-workflow-table/search-workflow-table.component.html

```diff
@@ -33,16 +33,18 @@
       <mat-cell fxShow fxHide.lt-sm class="duration-cell" *matCellDef="let workflow">{{ workflow?.author || 'n/a' }}</mat-cell>
     </ng-container>
     <ng-container matColumnDef="descriptorType">
-      <mat-header-cell fxShow fxHide.lt-sm data-cy="descriptorTypeHeader" *matHeaderCellDef mat-sort-header>Format</mat-header-cell>
-      <mat-cell fxShow fxHide.lt-sm data-cy="descriptorType" *matCellDef="let workflow">
+      <mat-header-cell fxShow fxHide.lt-sm data-cy="descriptorTypeHeader" *matHeaderCellDef mat-sort-header fxFlex="20%"
+        >Format</mat-header-cell
+      >
+      <mat-cell fxShow fxHide.lt-sm data-cy="descriptorType" *matCellDef="let workflow" fxFlex="20%">
         <div>
           <span class="nonclick-badge">{{ workflow?.descriptorType | uppercase }}</span>
         </div>
       </mat-cell>
     </ng-container>
     <ng-container matColumnDef="projectLinks">
       <mat-header-cell fxShow fxHide.lt-md *matHeaderCellDef>Project Links</mat-header-cell>
-      <mat-cell fxShow fxHide.lt-md *matCellDef="let entry" class="align-languages">
```

Comment:
Is this the only use of the class? If so, remove the class.

Reply:
@garyluu I need to use media query on this, fxLayout.lt doesn't work in this case for some reason...